web-marketplace/src/components/organisms/ChooseCreditsForm/ChooseCreditsForm.constants.tsx

```diff
@@ -3,3 +3,4 @@ import { msg } from '@lingui/macro';
 export const MAX_AMOUNT = msg`Amount cannot exceed`;
 export const MAX_CREDITS = msg`Credits cannot exceed`;
 export const POSITIVE_NUMBER = msg`Must be positive`;
+export const NOT_ENOUGH_BALANCE = msg`You don’t have enough`;
```

Comment:
aren't we supposed to display `You don’t have enough ${displayDenom}`?

Reply:
yes, that's what we do here: https://github.com/regen-network/regen-web/pull/2501/files#diff-dc24cb13d9ae4f19768619f0203bc0cbe44374ee901e3edb0bea0e4d6114a8e9R157

Reply:
oh I see, that assumes that all errors for this field will have the displayDenom at the end of the message.
I guess that works for now but that's not very reusable and also it gives less context for translating the content which could result in wrong translations.